app/selectworkspace/selectWorkspace.tsx

```diff
@@ -29,6 +33,14 @@ function SelectWorkSpace() {
     }
   }, []);
 
+  const handleClick = useCallback(async (workSpaceId: string) => {
+    // set current workspace in localstorage
+    UserPreferenceSingleton.getInstance().setCurrentWorkspace(workSpaceId);
+    // get workspace data
+    const result = await getWorkspaceById(workSpaceId);
```

Comment:
Here will change logic
 we will create new service inside that will call this service
We will also reset all store to values of ticket store

Reply:
Done